### DISCUSSION TITLE
mouse cortex data (GSE210749)

### DISCUSSION THREAD

**Discussion by nouroddinc** [aertslab/scenicplus#118] [Q&A]
Hi scenic plus team,
thank you so much for this productive and useful package.
Can you please help me to find mouse cortex data, mentioned in the preprint [paper](https://www.biorxiv.org/content/10.1101/2022.08.19.504505v1.full) as "GSE210749". In manuscript says that "10x multiome in the mouse cortex and scATAC-seq in the Drosophila eye disc are available in GEO (GSE210749)" but I couldn't find it.

Best,
Noori

**Comment by SeppeDeWinter** [comment]
Hi @nouroddinc 

Thank you for your interest.

This data will become publicly available upon publication of the paper (hopefully soon ;) ).

Best,

Seppe
